### DISCUSSION TITLE
Linux audio software matrix

### DISCUSSION THREAD

**Discussion by spamatica** [muse-sequencer/muse#1130] [Team Posts]
Stumbled on this, did a few updates, some things I'm not sure what they are meant to be. 
https://docs.google.com/spreadsheets/d/1n6MaGigGGTgFvoqBCMgt-GuSmy8nqAUfF8ryL4nJbS8/edit#gid=0

Could be good to keep up to date

**Comment by terminator356** [comment]
That has to be one of the most comprehensive feature lists I've seen. Well done. Bookmarked.
Quick note: Caught this on first glance: Under Soundfont Formats:
We now support SF3 via our MESS fluidsynth as of like, last week ;-)

**Comment by terminator356** [comment]
Oh, wait, it's for an actual release, not developer branches, sorry.

**Comment by kybos** [comment]
"Event editor (simple table)" should be green as well? I guess the classical Event list is meant here, which has been available in MusE since long?
